README.md

```diff
@@ -1 +1,48 @@
-# AI Application.
\ No newline at end of file
+<div align="center">
+    <h1>AI project</h1>
+    <img alt="Demo screenshot" src="repo_conf/demo.jpg" width="400"/> 
+    <h3><a href="https://www.google.com">Live demo</a></h3>
+</div>
+
+<br>
+<p>This is a website based on Flask used for hosting our AI models and implementing the full life cycle for them.</p>
```

Comment:
This is a Flask-based website for hosting...

Reply:
...and implementing the full model life cycle for them

Model life cycle is a keyword for recruiters